### ISSUE TITLE
yolov3 learning but not detecting

### ISSUE THREAD

**Issue by JoltonDsouza** [pjreddie/darknet#1763]
I am training on a dataset with ~90k images(640 x 640) with just a single class(person), running kmeans to find its own anchors, making required changes in filters in the .cfg file and changing class number/names in the .data/.classes files and run for an iteration of 18k with the following status:

![Screenshot from 2019-08-08 11-49-53](https://user-images.githubusercontent.com/36086002/62729574-10a50900-b9d3-11e9-8cf0-1022cf70cffe.png)

So, the training starts at ~6000 and learns till I achieve a loss of ~0.7 and I stop training. But when I am trying to test it out on a sample video, it does not detect any person. 
This is what my .cfg has,

batch=64
subdivisions=16
width=640
height=640
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.001
burn_in=500
max_batches=40000
policy=steps
steps=30000
scales=.1

What is exactly wrong?

**Comment by iibrahimli** [comment]
I have the same issue, please let me know if you resolve it

**Comment by bamwani** [comment]
Duplicate of #1521

**Comment by Adithia99** [comment]
just make wrong script or something
